2.13/2.13.3/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.13.3
+Напишите цикл, который предлагает prompt ввести число, большее 100. Если посетитель ввёл другое число – попросить ввести
+ещё раз, и так далее.
+Цикл должен спрашивать число пока либо посетитель не введёт число, большее 100, либо не нажмёт кнопку Отмена (ESC).
+Предполагается, что посетитель вводит только числа. Предусматривать обработку нечисловых строк в этой задаче необязательно.
+<script>
+	let i = 0;
+	while (i <= 100) {
+		i = +prompt('Введите число');
+		if (!i) break;
+		i++;
```

Comment:
If the user enters '100', the value i will be increased and will equal to '101' - the cycle will stop, but it shouldn't do it.

And what about a name of this variable?